store/storetest/channel_store.go

```diff
@@ -2494,8 +2497,10 @@ func testChannelStoreAutocompleteInTeamForSearch(t *testing.T, ss store.Store, s
 	o5.Type = model.CHANNEL_PRIVATE
 	store.Must(ss.Channel().Save(&o5, -1))
 
-	store.Must(ss.Channel().CreateDirectChannel(u1.Id, u2.Id))
-	store.Must(ss.Channel().CreateDirectChannel(u2.Id, u3.Id))
+	_, err := ss.Channel().CreateDirectChannel(u1.Id, u2.Id)
```

Comment:
This fixes the build. You also have the fix the same error in L2639
```suggestion
	_, err = ss.Channel().CreateDirectChannel(u1.Id, u2.Id)
```